src/main/fc/fc_tasks.c

```diff
@@ -366,13 +388,19 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .staticPriority = TASK_PRIORITY_HIGH,
     },
 
-    [TASK_BATTERY] = {
-        .taskName = "BATTERY",
-        .taskFunc = taskUpdateBattery,
-        .desiredPeriod = TASK_PERIOD_HZ(50),        // 50 Hz
+    [TASK_BATTERY_ALERTS] = {
+        .taskName = "BATTERY_ALERTS",
+        .taskFunc = taskBatteryAlerts,
+        .desiredPeriod = TASK_PERIOD_HZ(5),        // 5 Hz
         .staticPriority = TASK_PRIORITY_MEDIUM,
     },
 
+    [TASK_BATTERY_SENSORS] = {
```

Comment:
there is quite a difference:

OLD:
```
arm-none-eabi-size ./obj/main/cleanflight_SPRACINGF3NEO.elf
   text	   data	    bss	    dec	    hex	filename
 249492	   6464	  34304	 290260	  46dd4	./obj/main/cleanflight_SPRACINGF3NEO.elf
arm-none-eabi-objcopy -O ihex --set-start 0x8000000 obj/main/cleanflight_SPRACINGF3NEO.elf obj/cleanflight_2.0.0_SPRACINGF3NEO.hex
```

NEW: (1 extra task)
```
arm-none-eabi-size ./obj/main/cleanflight_SPRACINGF3NEO.elf
   text	   data	    bss	    dec	    hex	filename
 249492	   6464	  34304	 290260	  46dd4	./obj/main/cleanflight_SPRACINGF3NEO.elf
arm-none-eabi-objcopy -O ihex --set-start 0x8000000 obj/main/cleanflight_SPRACINGF3NEO.elf obj/cleanflight_2.0.0_SPRACINGF3NEO.hex
```

see ae9ae5ad355bc1988920d4d092892533c58284a4